pkg/util/admission/work_queue.go

```diff
@@ -51,6 +51,20 @@ var KVAdmissionControlEnabled = settings.RegisterBoolSetting(
 	"when true, work performed by the KV layer is subject to admission control",
 	true).WithPublic()
 
+// KVBulkOnlyAdmissionControlEnabled controls whether user (normal and above
+// priority) work is subject to admission control. If it is set to true, then
+// user work will not be throttled by admission control but bulk work still will
+// be. This setting is a preferable alternative to completely disabling
+// admission control. It can be used reactively in cases where index backfill,
+// schema modifications or other bulk operations are causing high latency due to
+// io_overload on nodes.
+// TODO(baptist): Find a better solution to this in v23.1.
+var KVBulkOnlyAdmissionControlEnabled = settings.RegisterBoolSetting(
+	settings.SystemOnly,
+	"admission.kv.bulk_only.enabled",
+	"when both admission.kv.enabled and this is true, only throttle bulk work",
+	false).WithPublic()
```

Comment:
Drop the .WithPublic(). We'll only use it selectively in escalations. It doesn't need to make its way into release notes or such, which brings with it more irrelevant backward compat considerations.